docker-compose.yml

```diff
@@ -1,20 +1,22 @@
-# Use postgres/example user/password credentials
-version: '3.1'
+# create a docker image with mysql and a default password of admin
+version: "3"
+
+name: dotfyle 
 
 services:
   db:
-    image: postgres
+    image: bitnami/postgresql:latest
+    container_name: dotfyle-db
     restart: always
-
-    ports:
-      - 5432:5432
-
     environment:
-      POSTGRES_USERNAME: postgres
-      POSTGRES_PASSWORD: postgres
-
-  adminer:
-    image: adminer
-    restart: always
+      POSTGRESQL_USERNAME: postgres
+      POSTGRESQL_PASSWORD: postgres 
+      POSTGRESQL_DATABASE: dotfyle_dev 
+    # create a volume for the docker container_name
+    volumes:
+      - dotfyle-data:/bitnami/postgres/data
     ports:
-      - 8080:8080
```

Comment:
i prolly should use the port you had here

Reply:
why did you update the docker-compose file?

Reply:
Because it's easier to use bitmami image to create the DB https://github.com/codicocodes/dotfyle/commit/cbdfe1bf531ecb5be1b4aa0d495b5594c49fd40f